JakinChan200/CS180_Roald
# Deleted duplicates || likes/dislikes/comment == 0

## Overview
Got rid of duplicate entries
- Entries with the same video ID (Everything else can be different)
Got rid of Entries where the amount of Likes, Dislikes, and Comments are 0

## Materials
 - Issue #29 

## Pictures
 
![image](https://user-images.githubusercontent.com/42459094/164135107-79f22003-c7f0-473d-9236-9b0e8e3810a6.png)


## Manual Testing Guide
- Navigate to the countries pages
- Navigate to the search bar and click it.
- Click a category.
- Note that the results do not have more than one of each
